jobs/build/ocp/Jenkinsfile

```diff
@@ -452,30 +493,80 @@ node(TARGET_NODE) {
               )
             }
 
+            // TODO: Remove after new OIT version confirmed
+            // Old method for most images
             stage( "compare dist-git" ) {
                 sh "ose_images.sh --user ocp-build compare_nodocker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base"
             }
 
             stage( "update dist-git" ) {
                 sh "ose_images.sh --user ocp-build update_docker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --force --release '${NEW_DOCKERFILE_RELEASE}' --version 'v${NEW_VERSION}'"
             }
+            // End old method
+
+            // OIT Method
+            OIT_WORKING = "${pwd(tmp=true)}"
+
+            stage( "update dist-git" ) {
+              buildlib.write_sources_file()
+              buildlib.oit """
+--working-dir ${OIT_WORKING} --group 'openshift-${BUILD_VERSION}' \\
+distgits:rebase --sources ${env.WORKSPACE}/sources.yml --version ${NEW_VERSION} \\
+--release ${NEW_DOCKERFILE_RELEASE}
+"""
+            }
+
+            record_log = buildlib.parse_record_log( OIT_WORKING )
+            distgit_notify = buildlib.get_distgit_notify( record_log )
+
+            for(kv in buildlib.mapToList(distgit_notify)) {
```

Comment:
Needs comments. Does this 'for in' actually run in Jenkins?

Reply:
Pulled from SO questions about Jenkins/Groovy, but easy enough to make it a normal for loop which is obviously safer.

Reply:
I'm fine with it if it works. I've just had trouble with 'in' with Jenkins.